api-server/src/server/boot/certificate.js

```diff
@@ -217,7 +217,8 @@ function getUserIsCertMap(user) {
     isFullStackCert = false,
     isSciCompPyCertV7 = false,
     isDataAnalysisPyCertV7 = false,
-    isMachineLearningPyCertV7 = false
+    isMachineLearningPyCertV7 = false,
+    isRelationalDatabasesCertV8 = false
```

Comment:
Reminder to add this, at some point, to the `sendCertifiedEmail` function

Reply:
This is something easy to forget.  If it needs to happen we should create a draft PR.